### ISSUE TITLE
Admin Labels and Links Are Prefixed

### ISSUE THREAD

**Issue by joshvickerson** [anchorcms/anchor-cms#1142]
### Summary

Not really sure how to really explain this bug but... The admin page's labels, links and actions are all "prefixed" in a sense. The below screenshots show the problem... I'm not sure how else to explain it. It's not just on these areas, but on every single label or placeholder text in the admin UI.

![screen shot 2016-08-10 at 8 11 16 pm](https://cloud.githubusercontent.com/assets/2325487/17575491/9001c6ba-5f37-11e6-9f7f-42082a24a9be.png)
![screen shot 2016-08-10 at 8 11 38 pm](https://cloud.githubusercontent.com/assets/2325487/17575493/900b987a-5f37-11e6-87a3-6573e4d7acbb.png)

### Expected Behaviour

These weird prefixes should not be there.

### Actual Behaviour

The labels are a bit nonsensical.

### Context details (if applicable)

- Anchor version: 0.12.1
- Server setup: Ubuntu 16.04, Apache 2.4.23, PHP 7.0.9, MySQL 5.7.13
- URL: blog.joshvickerson.com

**Comment by Gerrit0** [comment]
This is caused by a missing translation file - what did you choose when installing Anchor? 

Simple fix: 

Copy [these files](https://github.com/anchorcms/anchor-cms/tree/master/anchor/language/en_GB) to the corresponding folder on your site. (If they don't already exist)

In `anchor/config/app.php` change the language setting to `en_GB`

**Comment by daviddarnes** [comment]
Thanks @Gerrit0, this is exactly the case. I'm guessing this might have been a language that was removed in a later version of Anchor, so it might be worth digging out the one you want from the translations repo https://github.com/anchorcms/anchor-translations

**Comment by joshvickerson** [comment]
Ah, okay. That makes sense. I had edited the language setting because the language codes (apparently) aren't valid for RSS feeds (according to a feed validator I used), so I had changed it to just the two character code "en".

Not that the RSS feed passing validation is a *big* deal, but would there be an acceptable solution to handling this? Maybe have the RSS feed just use the first two characters of the language setting? If this is something we want to solve I can submit a PR.
